### PR TITLE
Added iOS Link & Presentation Mode

### PR DESCRIPTION
**Pull request by YewoMhango**
For issues #71 and #72

### CODE REVIEW

**Comment by TinasheMzondiwa** [comment]
🙇🏾‍♂️ Great work.
Looks like the presentation mode doesn't work on mobile, I tested on iOS 17 safari browser.
If we can't get it to work on mobile can we hide the button instead?

**Comment by YewoMhango** [comment]
> 🙇🏾‍♂️ Great work.
> Looks like the presentation mode doesn't work on mobile, I tested on iOS 17 safari browser.
> If we can't get it to work on mobile can we hide the button instead?

Oh, okay, I see. It's working for me on Android, so I think the problem isn't mobile in general. Let me try and see what could be the issue

**Comment by YewoMhango** [comment]
@TinasheMzondiwa so it does indeed seem that Safari on iOS doesn't currently support going fullscreen. I've added a check which only displays the fullscreen button when the user's browser supports it. You should verify whether that works as expected

File: src/Pages/SongPage/PresentationMode/PresentationMode.tsx [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+    useEffect(() => {
+        const [type, content] = selectedHymn.content
+            ? ["html", selectedHymn.content]
+            : selectedHymn.markdown
+            ? ["markdown", selectedHymn.markdown]
+            : [null, ""];
+
+        if (type) {
+            const formattedVerses =
```
[the remark this answers is not in the document]
Comment (YewoMhango) [reply src/Pages/SongPage/PresentationMode/PresentationMode.tsx]: Okay, I've just done that

**Review by TinasheMzondiwa — APPROVED** [review]

**Pull request merged**